providers/oldTrello.js

```diff
@@ -0,0 +1,557 @@
+// trello.js
```

Comment:
Keeping this file as a record until I finish testing the new provider. Once that's complete it will be deleted.